highlight/highlight
# Fix state reset on session transition

## Summary

In Waterloo last week we noticed that the view mode on sessions would get out of sync when transitioning sessions. e.g. you could be in comment mode, but when you click a session card in the feed you were transitioned to inspect mode and the session right panel was opened while the UI still showed you being in comment mode. We always want to show the comments panel when you are in comment mode, so this PR removes the state reset when clicking on a session card.

Note that [there is still logic](https://github.com/highlight/highlight/blob/db9259f8bb9877036735fe3c3cfef66af16ad066/frontend/src/pages/Player/RightPlayerPanel/RightPlayerPanel.tsx#L62-L68) to reset to the session view if you are in the event view when transitioning sessions.

## How did you test this change?

* View a session
* Transition to comment mode
* View another session
* Ensure you are still in comment mode and that comment panel is open

## Are there any deployment considerations?

N/A

## Does this work require review from our design team?

N/A
